.travis.yml

```diff
@@ -11,8 +11,10 @@ matrix:
     # Restore matrix job entry from d2d67fab to test OSX
     - python: 2.7
       env: UNSUPPORTED=true
+      script: .misc/check_unsupported.sh
     - python: 3.3
       env: UNSUPPORTED=true
```

Comment:
Ya, unnecessary now, but helps people understand results on the build overview page.